exercises/luhn/example.jl

```diff
@@ -0,0 +1,16 @@
+function luhn(id::AbstractString)
+    id = split(replace(id, ' ', ""), "")
+    length(id) < 2 && return false
+    all(isdigit, id) || return false
```

Comment:
Lines 3-4 can be merged in one line with one return statement, but it will be harder to read.